app/code/community/Bolt/Boltpay/Helper/Api.php

```diff
@@ -189,12 +189,10 @@ public function createOrder($reference, $session_quote_id = null)
             throw new Exception("The Bolt order reference does not match the current cart ID.");
         }
 
-        $display_id = $transaction->order->cart->display_id;
+        $order_id = $transaction->order->cart->display_id;
```

Comment:
I wonder if we should call it what it is on the order and use increment_id, or what it is on the quote and call it reserved_order_id. When I think of order_id, I think of entity_id on sales_flat_order which isn't what order_id is here. I'm good either way here, but my preference would be to call it either reserved_order_id or increment_id.

Reply:
Good point.  How I look at the naming of the variable is and the line of code is to give as much context to the next developer as possible.  So, in this line of code, if there is a new developer, he or she will see that the `order_id` is equivalent to the Bolt concept of `display_id`.  However, I fail to convey to him that in Magento, the displayed id is actually the `increment_id` in Magento speak.  I believe the Magento poorly named this and is inherently confusing, so I propose calling the variable `$order_increment_id` so that all of the above ideas are conveyed.

Reply:
+1 
I think I like `reservedOrderId` or `incrementId`.

Reply:
+1 @daisy1754

Reply:
ok.  `reservedOrderId`, camel_cased seems like a winner.  Look close, there is a funny joke in there. :dancing_women: